scripts/scheduler.py

```diff
@@ -351,8 +354,24 @@ def queue_new_jobs(available_slots, env, config, studies, wes_token, exclude_stu
 
     # now queue and move the job one-by-one
     for job in jobs_to_queue:
-        # TODO: support resume, detect whether run info file exists, if so get session id
-        resume = False  # set resume to the session id, set to None for now
+        # support resume, detect whether run info file exists, if so get session id
+        exist_run_path = os.path.join(job, f'run.*.*.wes-*')
```

Comment:
the run info file name pattern needs to take into account the current `env`, something like `run.*.{env}.wes-*`. This way, don't need to deal with unrelated run info file. The idea is that a job will stick to the env that it previously ran, only in such case resume is possible